.github/workflows/on-main_release-drafter.yml

```diff
@@ -1,5 +1,4 @@
-name: Release Drafter
-
+name: "[on Main] Release Drafter"
```

Comment:
I'm not sure what this would give us (if possible, this release-drafter action was quite limited in its dynamic nature, hence the current setup with two configs)

once this PR is merged, and then 5.x.x is made, and the workflow changes _there_ are made, we will be getting 2 release drafts, one for main and one for 5.x.x. 

What would making this more dynamic net us?

Reply:
just more scaleable